src/renderer/App.vue

```diff
@@ -1,20 +1,60 @@
 <template>
-  <div id="app">
-    <landing-page></landing-page>
-  </div>
+  <main id="app">
+    <Carousel :perPage="1">
+      <Slide>
+        <LandingPage />
+      </Slide>
+      <Slide>
+        <div>Hello Tasks !!</div>
+      </Slide>
+    </Carousel>
+  </main>
 </template>
 
 <script>
+import { Carousel, Slide } from 'vue-carousel'
 import LandingPage from '@/components/LandingPage'
 
 export default {
   name: 'HomeAssistant',
   components: {
-    LandingPage
+    Carousel,
+    LandingPage,
+    Slide
   }
 }
 </script>
 
 <style>
-/* CSS */
+body {
+  margin: 0;
+}
+
+main {
+  width: 100%;
+  height: 100vh;
+  overflow: hidden;
+  background-color: black;
+}
+
+* {
+  color: white;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 300;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-family: 'Roboto Condensed', sans-serif;
+  font-weight: 500;
+}
+
+.VueCarousel-pagination {
```

Comment:
Maybe you could just use `paginationEnabled` Carousel props ? seen (here)[https://github.com/SSENSE/vue-carousel]

Reply:
I wwas looking for something like this :) Thanks !